src/WinAppDriver/IApplication.cs

```diff
@@ -11,5 +13,9 @@ internal interface IApplication
         void BackupInitialStates();
 
         void RestoreInitialStates();
+
+        void Uninstall();
+
+        void Install(string zipFile);

```

Comment:
Do not expose the source format in the interface. Maybe `source` is better than `zipFile`?